### PR TITLE
Cache getMetadata() Format

### PR DESCRIPTION
**Pull request by MGatner**
**Description**
`CacheInterface::getMetadata()` thus far returns a generic array of info "dependent on each adapter", as well as `false` or `null` for missing items (also dependent on each adapter). This PR attempts to standardize the expected return values while still leaving the `false` in place for now. Additionally:
* Fixes a bug where "expire" was not always an absolute time value
* Fixes a discrepancy between relative and absolute TTL values
* Adds support for `null` values to "expire" when a TTL was specified as "never expire"

**Checklist:**
- [X] Securely signed commits
- [X] Component(s) with PHPdocs
- [X] Unit testing, with >80% coverage
- [X] User guide updated
- [X] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/changelogs/v4.1.2.rst [hunk 1]
```diff
@@ -26,6 +26,7 @@ Changes:
 - ``Entity::mutateDate`` uses external cast handler ``DatetimeCast::get``.
 - In order for ``Config\**`` classes to get their respective properties' values from the ``.env``, it is now necessary to namespace the property with the name of the class. Previously, the property names are enough but now disallowed because it can get system environment variables, like ``PATH``.
 - The array helper ``_array_search_dot`` is now marked for ``@internal`` use. As this is used by ``dot_array_search``, users should not use ``_array_search_dot`` directly in their code.
+- ``CacheInterface::getMetaData()`` returns have been specified to be null or and array with at least the "expires" key (absolute epoch expiration); on misses the File, Memcached, and Wincache Handlers still return ``false`` which will become ``null` in a future release.
```
Comment (paulbalandan) [inline user_guide_src/source/changelogs/v4.1.2.rst:29]: UG check fails here

Comment (MGatner) [reply user_guide_src/source/changelogs/v4.1.2.rst:29]: I don't see it on mobile. "User Guide syntax check: successful in 1m 14s". Do you know what the problem is?

Comment (MGatner) [reply user_guide_src/source/changelogs/v4.1.2.rst:29]: Ohhh I see I missed a backtick

Comment (MGatner) [inline user_guide_src/source/changelogs/v4.1.2.rst:29]:
```suggestion
- ``CacheInterface::getMetaData()`` returns ``null`` for misses, or an array with at least the "expires" key (absolute epoch expiration). The File, Memcached, and Wincache Handlers still return ``false`` which will become ``null`` in a future release.
```

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**